apps/web/cypress/e2e/multi-dwelling/multi-dwelling-9.9.9-walkthrough.cy.ts

```diff
@@ -30,6 +38,26 @@ describe("multi dwelling: 9.9.9 walkthrough", () => {
     ).should("be.hidden");
   });
 
+  it.only("validate p18 shows value from previous question p5", () => {
+    // This walkthrough should hit both P5 and P18
+    const walkthroughContainingP5P18 = getWalkthrough(walkthroughs, 1);
+    let p5AnswerValue = "";
+
+    for (const step of walkthroughContainingP5P18.steps) {
```

Comment:
made this a for loop instead of a forEach because `break` statements don't work otherwise